### PR TITLE
Move RawURL to the front of Github

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: conda_forge_tick/update_upstream_versions.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def get_latest_version(meta_yaml, sources):
     return False
 
 
-def update_upstream_versions(gx, sources=(PyPI(), CRAN(), Github(), RawURL())):
+def update_upstream_versions(gx, sources=(PyPI(), CRAN(), RawURL(), Github())):
```
Comment (CJ-Wright) [inline conda_forge_tick/update_upstream_versions.py]: Will this increase execution times?

Comment (isuruf) [reply conda_forge_tick/update_upstream_versions.py]: Probably. Are the 4 scripts run separately or all at once. If separately, I don't think it'll be a huge issue.

Comment (CJ-Wright) [reply conda_forge_tick/update_upstream_versions.py]: They are currently run in sequence.

**Comment by scopatz** [comment]
Any objections to this order?

**Comment by CJ-Wright** [comment]
@isuruf would it be possible to use asyncio here? (discussing with @scopatz would it be possible to stuff each source into its own thread)

**Comment by CJ-Wright** [comment]
LGTM

**Pull request merged**